components/SlateRenderer/SlateRenderer.tsx

```diff
@@ -29,6 +32,7 @@ const components: ComponentRenderers = {
     // [LINK_NODE_TYPE]: ({ children, node }) => <a href={node.href}>{children}</a>,
     [PARAGRAPH_NODE_TYPE]: ({ children }) => <p>{children}</p>,
     // [QUOTE_NODE_TYPE]: ({ children }) => <blockquote>{children}</blockquote>,
+    [CONTACT_NODE_TYPE]: ({ node }) => <Contact node={node} className={styles.contact} />,
```

Comment:
I don't know how you didn't manage to get it to work with `not-prose` class. It just worked for me first try:
```suggestion
    [CONTACT_NODE_TYPE]: ({ node }) => <Contact node={node} className="not-prose" />,
```

This is the only change required for the fix, so the stylesheet is not needed.